content/en/docs/components/serving/bentoml.md

```diff
@@ -0,0 +1,258 @@
++++
+title = "BentoML"
+description = "Model serving with BentoML"
+weight = 40
++++
+
+{{% stable-status %}}
+
+## Model serving with BentoML
+
+In this guide, we will demo serving a scikit-learn based iris classifier model with
```

Comment:
A technical writing hint: It's better to avoid the first person ("we") because the reader can become confused. It's not clear who "we" is in this context.

Suggestion:

```
This guide demonstrates how to serve a scikit-learn...
```

Reply:
Thanks for the tips. I am still very new at technical writing, love to get more feedback on how I can improve it